test/mailers/previews/notifier_preview.rb

```diff
@@ -0,0 +1,7 @@
+class NotifierPreview < ActionMailer::Preview
```

Comment:
@sergeyradkevich, this is a built in way of email previewing. Using it I can preview email template on a test route.